mono/monodevelop
# Implemented '546162 - Generate overrides code fix ui'

https://devdiv.visualstudio.com/DevDiv/_workitems?id=546162&_a=edit

Test instructions : 

* Place caret inside a new class:

class Test { 
  |
}

* Right click -> Quck Fix -> select  Generate overrides...
* Select overrides & hit ok -> the overriden members should be created
* Select cancel -> nothing should happen
